src/pybind/mgr/dashboard/controllers/host.py

```diff
@@ -375,6 +375,12 @@ def get(self, hostname: str) -> Dict:
 
     @raise_if_no_orchestrator([OrchFeature.HOST_LABEL_ADD, OrchFeature.HOST_LABEL_REMOVE])
     @handle_orchestrator_error('host')
+    @EndpointDoc('',
+                 parameters={
+                     'hostname': (str, 'Hostname'),
+                     'labels': ([str], 'Host Labels'),
+                 },
+                 responses={200: None})
```

Comment:
Also a valid repsonse status can be ```204 (No Content)``` when no response is returned.

Reply:
Ok, will add it too.